code/modules/hydroponics/grown/melon.dm

```diff
@@ -39,6 +40,36 @@
 /obj/item/food/grown/watermelon/make_dryable()
 	return //No drying
 
+/obj/item/food/grown/watermelon/attackby(obj/item/I, mob/user, params)
+	if(!istype(I, /obj/item/kitchen/spoon))
+		return ..()
+
+	var/melon_pulp_count = 1
+	if(seed)
+		melon_pulp_count += round(seed.potency / 25)
+
+	user.balloon_alert(user, "scooped out [melon_pulp_count] pulp")
+	for(var/i in melon_pulp_count)
+		new /obj/item/food/watermelonmush(user.loc)
+
+	var/melon_armour
```

Comment:
you should type this variable to be what you would expect (in this case clothing I think?)

Reply:
well I expect nothing, until it can be defined. Why would I make it something if it's gonna be changed rampantly?

Reply:
melon_armor is always going to be some sort of clothing, right? it's useful to make that clear so people are more able to reason about what it is